src/status_im/contexts/settings/wallet/keypairs_and_accounts/import_seed_phrase/view.cljs

```diff
@@ -0,0 +1,76 @@
+(ns status-im.contexts.settings.wallet.keypairs-and-accounts.import-seed-phrase.view
+  (:require
+    [quo.core :as quo]
+    [react-native.core :as rn]
+    [react-native.safe-area :as safe-area]
+    [reagent.core :as reagent]
+    [status-im.common.enter-seed-phrase.view :as enter-seed-phrase]
+    [status-im.common.standard-authentication.core :as standard-auth]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn import-seed-phrase-controls
+  [{:keys [submit-disabled?
+           container-style
+           prepare-seed-phrase
+           seed-phrase
+           set-invalid-seed-phrase
+           focus-input]}]
+  (let [keypair               (rf/sub [:get-screen-params])
+        customization-color   (rf/sub [:profile/customization-color])
+        show-errors           (rn/use-callback
+                               #(js/setTimeout
+                                 (fn []
+                                   (focus-input)
+                                   (reagent/next-tick set-invalid-seed-phrase))
+                                 100))
```

Comment:
Here we're a small timeout to wait for the bottom-sheet to be hidden before trying to focus form input and show the keyboard with validation errors.